### DISCUSSION TITLE
[Accounts] Currently URL for accounts.ente.io is hardcoded

### DISCUSSION THREAD

**Discussion by AlexanderS** [ente-io/ente#3202] [Enhancements]
### Description

I tried to self-host the accounts webapp for passkey management, but it seems that currently the URL for this webapp is hardcoded in the different clients, even when connecting to a custom API server. 

Ideally it would be possible to configure the URL for this webapp in the museum server, so that the clients can automatically retrieve the correct URL. 

### Version

v3.1.3

### What product are you using?

Ente Auth

### What platform are you using?

Mobile - Android, Desktop - Linux, Web - Chrome, Server

**Comment by mngshm** [comment]
The accounts endpoint is only hardcoded in the mobile and desktop app. Checkout `ente/web/apps/photos/.env` and `setup the NEXT_PUBLIC_ENTE_ACCOUNTS_URL` to the accounts app, and it should be working fine.

**Comment by ua741** [comment]
You can now configure this url in your server config. [Ref](https://github.com/ente-io/ente/blob/543411254b2bb55bd00a0e515dcafa12d12d3b35/server/configurations/local.yaml#L76-L89)
